ansible/roles/test/files/ptftests/arista.py

```diff
@@ -401,8 +401,8 @@ def parse_bgp_neighbor_once(self, output):
         is_gr_ipv6_enabled = False
         restart_time = None
         for line in output.split('\n'):
-            if '     Restart-time is' in line:
-                restart_time = int(line.replace('       Restart-time is ', ''))
+            if 'Restart-time is' in line:
+                restart_time = int(re.search(r'\d+', line).group())
```

Comment:
just a question, can this change back compatible with older version of eos? @andywongarista

Reply:
Yes, only difference from older version of eos is the indentation. New logic just ignores indentation and matches on the time value.